packages/trace-viewer/.gitignore

```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+public/sw.bundle.js*
```

Comment:
Why `*` at the end? What else could this be, except for `sw.bundle.js`?

Reply:
it also emits `sw.bundle.js.map`, and I was too lazy to spell that out. but if you're already asking about this, it makes sense to spell it out I guess. will do

Reply:
scratch that - it used to emit the source map, but doesn't anymore. i've removed the *